ekncontent/ekncontent/eknc-article-object-model.c

```diff
@@ -321,6 +347,24 @@ eknc_article_object_model_get_authors (EkncArticleObjectModel *self)
 }
 
 /**
+ * eknc_article_object_model_get_temporal_coverage:
+ * @self: the model
+ *
+ * Get the temporal coverage over the article.
+ *
+ * Since: 2
```

Comment:
I found out that once you have Since: annotations, the versioned API index must be included in `eos-knowledge-content-docs.xml` as well:
> /home/philip/.cache/jhbuild/build/eos-knowledge-lib/ekncontent/docs/reference/ekncontent/eos-knowledge-content-docs.xml doesn't appear to include "xml/api-index-2.xml"

Please make sure to run the tests with gtk-doc enabled before submitting this for review again, in order to cut down the turnaround.